prql-compiler/src/ast/item.rs

```diff
@@ -301,6 +306,11 @@ impl Display for Item {
                 for arg in &func_call.args {
                     write!(f, " {}", arg.item)?;
                 }
+
+                match func_call.name.as_str() {
```

Comment:
This is the closing parenthesis around aggregates, see more detailed comment above.